.env.example

```diff
@@ -0,0 +1,12 @@
+DB_DATABASE=postgres
+DB_HOST=localhost
```

Comment:
<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(36, 41, 47); font-family: -apple-system, &quot;system-ui&quot;, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">To show an env example you can use Readme.md file, and put something like that:</p>

Variable Name | Description
-- | --
DB_HOST | Database host name
DB_PORT | Database port.
DB_NAME | Database name.
DB_MIGRATE | Migrate database when the app is runing.
DB_USERNAME | Username used to establish the connection.
DB_PASSWORD | Password used to establish the connection.

<br class="Apple-interchange-newline">

Reply:
Nice idea!! Thanks